flutter_news_example/lib/login/widgets/login_form.dart

```diff
@@ -14,7 +14,7 @@ class LoginForm extends StatelessWidget {
     final l10n = context.l10n;
     return BlocListener<AppBloc, AppState>(
       listener: (context, state) {
-        if (state.status == AppStatus.authenticated) {
+        if (state.status.isLoggedIn) {
```

Comment:
Can we add tests for these updates? It'd be nice to prevent an regressions in the future, thanks! 🙏

Reply:
@felangel @marwfair Tests added 🧪